src/InlineObjectCreator/widget/InlineObjectCreator.js

```diff
@@ -159,37 +171,41 @@ mendix.widget.declare('InlineObjectCreator.widget.InlineObjectCreator', {
 		}
 		
 		this.feedback(this.savecaption);
-		mx.processor.createObject({
-			"className"	: this.entity,
-			"callback"	: dojo.hitch(this, this.created, input),
-			"context"	: null
+		mx.data.create({
+			entity: this.entity,
+			callback: lang.hitch(this, this.created, input)
 		});
 	},
 	
 	created : function(value, object) {
 		//update ownership
-		object.setAttribute(this.ownerattribute.split("/")[0], this.dataobject);
-		object.setAttribute(this.attribute, value);
+		object.set(this.ownerattribute.split("/")[0], this.dataobject);
+		object.set(this.attribute, value);
 		var me = this;
-		object.save({
-			error : dojo.hitch(this, this.feedbackError, "Error while saving object"),
-			callback : dojo.hitch(me, me.invokeMF, object)
-		});
+		if (object) {

```

Comment:
Why `if`? and why here? if it is a problem you need to it before the `object.set` and show an error to the user. Or at least to the console.